examples/one-page/index.html

```diff
@@ -13,7 +13,7 @@
     <script src="https://unpkg.com/react-dom@18.2.0/umd/react-dom.production.min.js"></script>
     <script src="https://unpkg.com/react-is@18.2.0/umd/react-is.production.min.js"></script>
     <script src="https://unpkg.com/prop-types@15.7.2/prop-types.min.js"></script>
-    <script src="https://unpkg.com/spectacle@^10/dist/spectacle.min.js"></script>
+    <script src="https://unpkg.com/spectacle@^10/lib/index.global.js"></script>
 <!--    <script src="../../packages/spectacle/dist/spectacle.min.js"></script>-->
```

Comment:
One teeny thing, change the comment on this path to match the new `lib/index.global.js` output locally